vm/runtime/src/lib.rs

```diff
@@ -182,9 +183,31 @@ pub trait Syscalls {
         Ok(data_commitment_v1_to_cid(&comm_d))
     }
     /// Verifies a sector seal proof.
-    fn verify_seal(&self, _vi: &SealVerifyInfo) -> Result<(), Box<dyn StdError>> {
-        // TODO
-        todo!()
+    fn verify_seal(&self, vi: &SealVerifyInfo) -> Result<(), Box<dyn StdError>> {
+        let commd = cid_to_data_commitment_v1(&vi.unsealed_cid)?;
+        let commr = cid_to_replica_commitment_v1(&vi.on_chain.sealed_cid)?;
+        let miner_addr = Address::new_id(vi.sector_id.miner);
+        let mut prover_id = ProverId::default();
+        prover_id.copy_from_slice(&miner_addr.payload_bytes());
```

Comment:
This will fail. The payload bytes is guaranteed to be under 32 bytes.

Reply:
Oh, you're right, didn't realize that function was panic, I was under the assumption that only panics if slice is longer

Reply:
Nice catch @ec2 ....hmmm so how should I go about this then? The go impl uses this [to32ByteArray](https://github.com/filecoin-project/filecoin-ffi/blob/870251cd04c54e7a3a08b714f3e71a9edec28445/proofs.go#L759) to make it 32 bytes.